moveos/smt/src/jellyfish_merkle/hash.rs

```diff
@@ -10,16 +16,12 @@ use primitive_types::H256;
 use proptest_derive::Arbitrary;
 use rand::{rngs::OsRng, Rng};
 use serde::{de, ser};
-use std::{
-    fmt::{self, Debug},
-    str::FromStr,
-};
 use tiny_keccak::{Hasher, Sha3};
 
 pub(crate) fn merkle_hash(left: HashValue, right: HashValue) -> HashValue {
     let mut value = left.to_vec();
     value.extend(right.to_vec());
-    HashValue::sha3_256_of(&value)
+    HashValue::sha256_of(&value)
```

Comment:
Bitcoin TagedHash

Reply:
> Bitcoin TagedHash

tagged hash is used for signing, in bitcoin merkel tree is double sha256:

sha256(sha256(data))